x-pack/platform/plugins/shared/triggers_actions_ui/public/plugin.ts

```diff
@@ -366,6 +366,7 @@ export class Plugin
           actionTypeRegistry,
           ruleTypeRegistry,
           share: pluginsStart.share,
+          enforceAdaptiveAllocations: !!pluginsStart.serverless,
```

Comment:
lets call this `isServerless`. Same for the rest

Reply:
Updated in 1e4dd5a78e1574dba4249fc9a7235f1f5a8b7bec